completion/hanjacnt.cwl

```diff
@@ -0,0 +1,175 @@
+# hanjacnt package
+# ponte-vecchio 2023-01-17 for v2.8
+
+#keyvals:\usepackage/hhanjacnt#c
+manspace
+20
+finhanja
+#endkeyvals
+
+#include: fontspec
+
+## 2.1 Numbers
+\NumHanja{%<number%>}
+\NumHangul{%<number%>}
+\NumHanjaBig{%<number%>}
+\NumHangulBig{%<number%>}
+
+## 2.2 Financial Hanja
+#ifOption:finhanja
+\FinHanjaMode
+\FinHanjaMode{%<option%>%keyvals%}
+#endif
+
+#keyvals\FinHanjaMode#
+full
+var
+fullvar
+off
+on
+#endkeyvals
+
+### 2.3 Spaces at 'man' (10,000) positions
+#ifOption:manspace
+\ManSpaceOn
+\ManSpaceOff
+#endif
+
+### 2.4 Digit
+\DigitHanja{%<number%>}
+\HanjaZero{%<char%>}
+\HanjaZeroFont{%<font%>}
+\TwentyHanjaChar[%<options%>%keyval]
+
+#keyvals\TwentyHanjaChar#c
+2
+10
+20
+30
+21
+31
+#endkeyvals
+
+### 2.5 Replace Hanja for a given number
+\MarkHanja{%<name%>%keyvals}{%<hanja%>}
+
+#keyvals\MarkHanja#c
+default
+one
+two
+three
+four
+five
+six
+seven
+eight
+nine
+ten
+hundred
+thousand
+shi
+bae
+chu
+man
+eok
+cho
+#endkeyvals
+
+## 2.6 Counters
+\HANJA{%<counter%>}#d*
```

Comment:
Do these define a command? Otherwise the `#d` does not belong here, and the trailing `*` doesn't do anything (I think). If you're trying to mark them unusual then you'd use `#*` here (or `#*d` if that were the correct spec)

Reply:
correct

Reply:
It converts the value of the counter being used to Hanja/Hangul, so probably not a `d` here. Reflected in the new commit.